content/en/docs/setup/platform-setup/gke/index.md

```diff
@@ -36,6 +36,20 @@ Follow these instructions to prepare a GKE cluster for Istio.
     the `gcloud container clusters create` command.
     {{< /warning >}}
 
+    {{< warning >}}
+    For private GKE Clusters, the firewall rule created does not open ports (9443, 15017) needed by validation webhooks.
```

Comment:
nit: 9443 is for older versions, I think we only document the latest. You should (?) only need 15017 now.

Reply:
Updated to 1.5. Pilot Discover port.